src/frontend/src/views/Data/components/TreesView/components/BlankStateTrees/style.ts

```diff
@@ -0,0 +1,77 @@
+import styled from "@emotion/styled";
+import {
+  CommonThemeProps,
+  fontBodyM,
+  fontHeaderM,
+  fontHeaderXl,
+  getSpaces,
+  Icon,
+  Link,
+} from "czifui";
+import { NewTabLink } from "src/common/components/library/NewTabLink";
+
+export const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-self: center;
+  ${(props: CommonThemeProps) => {
+    const spaces = getSpaces(props);
+    return `
+      margin-top: ${spaces?.xl}px;
+    `;
+  }}
+`;
+
+export const FlexColumn = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: 240px;
+`;
+
+const marginBottomL = (props: CommonThemeProps) => {
+  const spaces = getSpaces(props);
+  return `
+    margin-bottom: ${spaces?.l}px;
+  `;
+};
+
+export const Header = styled.h1`
+  ${fontHeaderXl}
+  ${marginBottomL}
+`;
+
+export const BodyText = styled.p`
+  ${fontBodyM}
+  ${marginBottomL}
+  margin-top: 0;
+`;
+
+const linkStyle = `
+  display: flex;
+  flex-direction: row;
+  cursor: pointer;
+`;
+
+export const StyledLink = styled(Link)`
+  ${linkStyle}
+  ${marginBottomL}
+`;
+
+export const StyledNewTabLink = styled(NewTabLink)`
+  ${linkStyle}
+`;
```

Comment:
Added the StyledNewTabLink because this one has one same tab and one new tab link on it.